tests/config/run_test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -xe
+
+if [ $# -eq 0 ]; then
+  echo "Error: Please path to the test binary path as a command-line argument."
+  exit 1
+fi
+
+TEMP_DIR="$(mktemp -d)"
+pushd "${TEMP_DIR}"
+
+function clean() {
+    rm -r "${TEMP_DIR}"
+}
+trap clean EXIT
+
+# Assign the base path from the argument
+binary_path=$1
+
+cp "${binary_path}/clvk.conf" "${TEMP_DIR}"
+
+# Run test
+CLVK_CONFIG_FILE="${binary_path}/conf_test.conf" \
```

Comment:
We probably want at least one env override to test that environment variables take precedence over config files.

Reply:
I think we are already doing so. As clvk.conf and conf_test.conf are both modifying the cache_dir configuration. Let me know if I am misunderstanding.